katz/src/main/kotlin/katz/Option.kt

```diff
@@ -125,6 +125,8 @@ sealed class Option<out A> {
     object None : Option<Nothing>() {
         override val isEmpty = true
     }
+
+
```

Comment:
Ejem! TwoBlankLinesException!! We need to add Detekt to the build system, it's official XD